server/src/Models.ts

```diff
@@ -0,0 +1,13 @@
+export enum playerState {
+  offline = 0,
+  waiting,
+  ready,
+}
+
+export interface Player {
+  id: string;
+  x: number;
+  y: number;
+  state?: playerState;
+  team?: number;
```

Comment:
`team` should only be available on `TD`. This means that you should extend `LobbyPlayer` to `TeamLobbyPlayer`(or something like that) and add the `team` prop.